shell/packages/sandstorm-db/db.js

```diff
@@ -966,7 +966,7 @@ SandstormDb = function (quotaManager) {
     settings: Settings,
     migrations: Migrations,
     // Omitted: StaticAssets
-    // Omitted: AssetUploadTokens
+    assetUploadTokens: AssetUploadTokens,
```

Comment:
So, in my ideal world `db.collections` wouldn't be a thing, an instead all of the collections would be private with specific methods on `SandstormDb` providing narrow access. This is why some collections were omitted here -- because the methods of `SandstormDb` already provided all the necessary operations on them, so I was trying to keep them private.

It looks like you're adding `assetUpdloadTokens` to `db.collections` here because you don't like referring to a global in the method below, but you aren't adding any external references. Is there a way we can solve both goals at once?